site/source/tutorials/008_animations/demo/initial/zombies/README.md

```diff
@@ -0,0 +1,26 @@
+# Animation Tutorial
+
+This project was generated with the [Dojo CLI](https://github.com/dojo/cli) & [Dojo CLI create app command](https://github.com/dojo/cli-create-app).
+For help setting up you development environment please see our [tutorial](https://dojo.io/tutorials/1050_development_environment/).
+
+## Build
+
+Run `dojo build` to create a production build for the project. The built artifacts will be stored in the `dist` directory.
+
+## Development server
+
+Run `dojo build -w` to create a development build and start a development server. By default the server runs on port `9999`, navigate to `http://localhost:9999/`.
```

Comment:
Same as above, the arguments have changed.

Reply:
Lol noticed this is the automatic README.md from the pre beta5 create app template! maybe just delete it.